ooniprobe/View/TestResults/TestSummaryViewController.m

```diff
@@ -85,22 +99,32 @@ - (void)reloadMeasurements {
         // If this test_name is not already a key in the dictionary, add it
         // with the current measurement as the value in an array
         if (!groupedMeasurements[testName]) {
-            groupedMeasurements[testName] = [NSMutableArray arrayWithObject:measurement];
+            groupedMeasurements[testName] = [@[measurement] mutableCopy];
         }
-        // If this test_name is already a key in the dictionary, append the current
-        // measurement to the existing array
+            // If this test_name is already a key in the dictionary, append the current
+            // measurement to the existing array
```

Comment:
This indenting was fine if we comment on the else. If you want to comment on the code below, then you should move the comment inside.